hs_core/hydroshare/users.py

```diff
@@ -595,7 +599,12 @@ def get_resource_list(creator=None,
 
             if user:
                 user = user_from_id(user)
-                queries[t].append(Q(edit_users=user) | Q(owners=user))
+                queries[t].append(Q(raccess__r2urp__privilege__lte=PrivilegeCodes.CHANGE,
+                                    raccess__r2urp__user=user.uaccess) |
+                                  Q(raccess__r2urp__privilege__lte=PrivilegeCodes.OWNER,
+                                    raccess__r2urp__user=user.uaccess)
+                                  )
+
```

Comment:
These are the kind of queries that should be *in* the new access control system itself. Doesn't it provide this mechanism for you?